src/functions.js

```diff
@@ -535,3 +535,19 @@ function latestPrice_ (slug, currency) {
 
   return result[currencyField]
 }
+
+function dailyClosingPrice_ (slug, day) {
```

Comment:
you can merge this after #43  and reuse the `priceOpenClose` (currently named `priceChange` function instead

Reply:
Hmm there are some differences: 
* one works with from/to, the other with a single day
* one is interested in just the closing price, the other in closing and opening price

```javascript
function priceChange_ (slug, from, to) {
  assertCanAccessHistoricData_(from)

  var endOfDay = beginningOfDaytoEndOfDay_(to)

  var results = getApiClient_().fetchOhlc(slug, from, endOfDay)
  assertHasData_(results)

  var firstDateResults = results[0]
  var lastDateResults = results[results.length - 1]

  if (firstDateResults == null ||
      lastDateResults == null ||
      !firstDateResults.hasOwnProperty('openPriceUsd') ||
      !lastDateResults.hasOwnProperty('closePriceUsd')) {
    throw new NoDataError_()
  }

  return {
    open: firstDateResults.openPriceUsd,
    close: lastDateResults.closePriceUsd
  }
}
```
```javascript
function dailyClosingPrice_ (slug, day) {
  assertCanAccessHistoricData_(day)
  var endOfDay = beginningOfDaytoEndOfDay_(day)

  var results = getApiClient_().fetchDailyClosingPrice(slug, day, endOfDay)
  assertHasData_(results)

  var result = results[0]

  if ( result == null || !result.hasOwnProperty('closePriceUsd')) {
    throw new NoDataError_()
  }

  return formatNumber_(result.closePriceUsd)
}
```

Reply:
in that case let's keep it